test/url_helpers_test.rb

```diff
@@ -11,6 +11,12 @@ class UniversesController < InheritedResources::Base
   defaults singleton: true, route_instance_name: 'universum'
 end
 
+class Galaxy < ModelBase
+end
+class GalaxiesController < InheritedResources::Base
+  singleton_belongs_to :universe
+end
+
```

Comment:
this is going to cover the missing line, but I do not understand if this makes sense

Reply:
Neither am I. I haven't used it before. I know there was an issue submitted to ActiveAdmin regarding this feature. I'm fine with this as is if it meets the coverage requirements. Adding an actual test for the feature can occur separately, it shouldn't delay this. Let me know what you prefer to do here.